tools/api-markdown-documenter/src/api-item-transforms/helpers/Helpers.ts

```diff
@@ -463,6 +463,7 @@ export function createRemarksSection(
 export function createThrowsSection(
 	apiItem: ApiItem,
 	config: Required<ApiItemTransformationConfiguration>,
+	title: string = "Throws",
```

Comment:
Nit: Can we call this `headingText` or something like that? That's more in line with the terminology used elsewhere in this helper suite. `wrapInSection` uses "title", but it has more particular semantics.